### DISCUSSION TITLE
Candidates for backporting to 3.4.x

### DISCUSSION THREAD

**Discussion by florian-h05** [openhab/openhab-webui#1816] [WebUI Maintainer Discussions]
These PRs are hot candidates for cherry-picking them to the 3.4.x branch:

I will only list PRs here, that have been tested in SNAPSHOTs (and some even the first milestone) for at least 2 wekks and no regressions were noticed in this time.

@openhab/webui-maintainers Can you please comment/vote?

**Comment by florian-h05** [comment]
MainUI Sitemap editor fixes:
- #1745 
- #1781 & #1783

/cc @mherwege

**Reply by florian-h05** [reply]
@ghys WDYT?

**Reply by florian-h05** [reply]
@kaikreuzer WDYT?

**Reply by kaikreuzer** [reply]
Sorry @florian-h05, I only saw your comment here after triggering the 3.4.3 build.
But having a glance at those, I'd rate them "nice to have" but nothing really critical that we are missing in 3.4.3.

**Reply by florian-h05** [reply]
Can we cherry pick them for 3.4.4?

**Reply by kaikreuzer** [reply]
I'd leave this to you and @ghys to decide.

**Comment by florian-h05** [comment]
MainUI fixes:

- [x]  #1670 (my PR, so I definitely need approval)
- [x]  #1765


To be considered in a few weeks after some extensive testing:
- #1807 
- #1808

**Reply by florian-h05** [reply]
@ghys WDYT? The last two PRs are there for future consideration, I wouldn’t bring them into 3.4.3 because I want them to be tested for at least 2 weeks.

**Reply by ghys** [reply]
I'm good with these 2 first PRs!

**Reply by florian-h05** [reply]
@ghys What do you think about backporting the third and fourth PRs to the 3.4.x branch? 
They are battle-tested since their merge and no problems were reported.

**Reply by ghys** [reply]
Sure. I've seen the big "connection failure" banner flash up on occasion when reloading the page, which doesn't look too good IMHO. Maybe it should be shown with a delay e.g. 1/2 seconds, not immediately.

**Reply by florian-h05** [reply]
I will take care of that.

**Reply by florian-h05** [reply]
#1993 implements a 1 second delay.
